package.json

```diff
@@ -12,8 +12,9 @@
     "npm": ">6.13.7"
   },
   "scripts": {
-    "test": "jest --coverage",
+    "test": "jest --coverage && npm run test:integration",
```

Comment:
what if we introduce also `test:unit` for `jest --coverage`?

while developing I often run `npm test` but now will stop as I do not want to run integration every time and best would be to have `npm run test:unit` too. We have a similar approach in the parser https://github.com/asyncapi/parser-js/blob/master/package.json

Reply:
Completely agree 👍  Changing!